src/globalComponents/SelectTank.js

```diff
@@ -0,0 +1,61 @@
+//@flow strict
+
+import * as React from 'react';
+
+type Props = {|
+	changeCurrentTank: (SyntheticInputEvent<HTMLInputElement>) => void
+|};
+
+type State = {|
+	tankList: Array<string>,

```

Comment:
I think it would be helpful to pass an array of Tanks around instead of just their names.